### DISCUSSION TITLE
will this app interfere with logi option+?

### DISCUSSION THREAD

**Discussion by kuyechao** [noah-nuebling/mac-mouse-fix#952] [Questions]
I have almost all functions setup nicely with logi option for my mx master. but I just want to use the function of holding a keyboard key and scroll for zooming. Would this app interfere with what I already set up with logi option+? If anyone has tried this, could you please let me know? Thanks!

**Comment by pandanimal** [comment]
I and another user are just discussing this in another thread, also with respect to logi option, and have an issue with the keyboard + scroll for zooming. Not working for me with logi pebble. Have yet to hear a response but hoping to.
